docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# 🌟 Apolusia Contributing Guidelines
+
+Thank you for your interest in contributing to Apollusia! We appreciate your effort and look forward to your contributions. Please take a moment to review these guidelines before getting started.
+
+## 📌 Important Note
+
+> **⚠️ IMPORTANT**
+>
+> **Pull Requests _without an associated issue_ will not be accepted. Please ensure you have an assigned issue before creating a Pull Request.**
+
+## 🚀 Prerequisites
+
+- 📚 **Open Source Etiquette**: New to open source? Read about [Basic etiquette](https://developer.mozilla.org/en-US/docs/MDN/Community/Open_source_etiquette) for open source projects.
+
+- 🐙 **Git and GitHub Basics**: Unfamiliar with these tools? Check out [GitHub for complete beginners](https://developer.mozilla.org/en-US/docs/MDN/Contribute/GitHub_beginners) for a comprehensive introduction.
+
+- 📦 **PNPM**: Make sure you have [PNPM](https://pnpm.io/installation) installed on your system.
+
+## ⚡️ Quick Setup
+
+### 🐳 Docker Compose
+
+Run Apollusia on your system using Docker Compose:
+
+```shell
+git clone --depth 1 --branch master https://github.com/Morphclue/apollusia 
+cd apollusia
+docker compose up -d
```

Comment:
For local development, it’s generally better to use pnpm commands for the frontend and backend, as it provides faster rebuild times and smoother debugging. So I would advise developers to only start the database / keycloak containers.

Reply:
@lcrojano 
I've added a new docker-compose file that can be used for dev-setup.
It can be started with `docker-compose -f docker-compose.dev.yml up` and after that `pnpm serve:frontend` and `pnpm serve:backend` should be used. Either with 3 terminals or 2 and a `-d` detach flag for compose.